query-containerd.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+for project in  $(gcloud projects list --format="value(projectId)")
+do
+    echo "ProjectId:  $project"
+    for clusters in $( \
+        gcloud container clusters list \
```

Comment:
I think this only returns clusters in a single location? Do you want to iterate all locations?

Reply:
it goes over all locations. I checked

Reply:
perhaps this doc needs to be updated: https://cloud.google.com/sdk/gcloud/reference/container/clusters/list